cloudformation/apps/aem-stack-manager/ssm-commands/AEM-ScheduleSnapshot.yaml

```diff
@@ -10,11 +10,11 @@ parameters:
       hours).
     type: String
     allowedPattern: ([1-9][0-9]{0,3})|(1[0-9]{1,4})|(2[0-7][0-9]{1,3})|(28[0-7][0-9]{1,2})|(28800)
-    default: '14400'
+    default: '3600'
```

Comment:
Are we sure about changing this back to 1 hour?

Reply:
Yes as this document is only for enabling/disabling the cronjob.